.github/dependabot.yml

```diff
@@ -109,3 +109,4 @@ updates:
     directory: /plugin/storage/cassandra
     schedule:
       interval: daily
+  
```

Comment:
need to add new config file

Reply:
I wanted to ask if should i add config file for opensearch and elastic search in this pr only will it be okay 
that  is why  I haven't added it yet

Reply:
does this config looks good 
```
- package-ecosystem: docker
    directory: /docker-compose/elasticsearch
    schedule:
      interval: daily
    allow:
      - dependency-name: "docker.elastic.co/elasticsearch/elasticsearch"
        update-types: ["version-update:semver-minor"]
```